scripts/build/Jenkins/Jenkinsfile

```diff
@@ -21,12 +21,9 @@ PIPELINE_TIMEOUT = 60
 
 EMPTY_JSON = readJSON text: '{}'
 
-ENGINE_DEFAULT_ORGANIZATION_NAME = 'o3de'
 ENGINE_REPOSITORY_NAME = 'o3de'
 ENGINE_DEVELOPMENT_BRANCH = 'development'
-EXTRAS_DEFAULT_ORGANIZATION_NAME = 'o3de'
-EXTRAS_REPOSITORY_NAME = 'o3de-extras'
-EXTRAS_DEVELOPMENT_BRANCH = 'development'
+ENGINE_URL = "https://github.com/o3de/o3de.git"
```

Comment:
If we hardcode the url here instead of loading it from scm.userRemoteConfigs.url, will this affect builds running from a branch in a fork? Meaning will the engine repo still come from o3de/o3de.git instead of fork/o3de.git

Reply:
ENGINE_URL is only used if the commit repo name is NOT o3de, in which case it pulls the development branch of o3de/o3de. If the commit repo is */o3de, meaning o3de/o3de or aws-lumberyard-dev/o3de or apmg/o3de, then it will use it.